server/handlers_test.go

```diff
@@ -48,3 +50,73 @@ func TestHandleHealthFailure(t *testing.T) {
 		t.Errorf("expected 500 got %d", rr.Code)
 	}
 }
+
+type emptyStorage struct {
+	storage.Storage
+}
+
+func (b *emptyStorage) GetAuthRequest(id string) (storage.AuthRequest, error) {
```

Comment:
[nit] We don't use the `b`, or the `id`, so this could be
```suggestion
func (*emptyStorage) GetAuthRequest(string) (storage.AuthRequest, error) {
```

Reply:
amended commit with this change